api/src/main/java/org/openmrs/module/chartsearch/ChartSearch.java

```diff
@@ -1,6 +1,6 @@
 /**
- * The contents of this file are subject to the OpenMRS Public License
```

Comment:
This as well as similar license headers should instead to totally changed to one defined in https://github.com/openmrs/openmrs-module-chartsearch/blob/master/api/src/main/java/org/openmrs/module/chartsearch/cache/ChartSearchPreference.java